### DISCUSSION TITLE
Changing Clock Speed

### DISCUSSION THREAD

**Discussion by wezleysherman** [jgilchrist/gbemu#16] [Ideas]
Is it possible to change the clock speed of the emulator? I'd like to make it runs at X2 speed.

**Comment by jgilchrist** [comment]
@wezleysherman I'm afraid there's not a particularly easy way to do this currently as the emulator syncs on video. I also haven't implemented targeting a specific FPS in the SDL implementation, it uses vsync so will most likely be around 60fps. If you use the SFML implementation you can change the target FPS to 120 which should have the effect of doubling the speed of the emulation.
